src/main/java/ch/wisv/chue/HueService.java

```diff
@@ -187,7 +198,7 @@ public void strobe(int millis, String... lightIdentifiers) {
             // Kinda magic symbolselection. It is something like this:
             // for 01..05 step 01, [0i0x]+ where i is `symbol` and x is light bulb
             JSONObject strobeJSON = new JSONObject();
-            strobeJSON.put("symbolselection", "01010301010102010301");
+            strobeJSON.put("symbolselection", "01010501010102010301040105");
```

Comment:
@jabbink I think this ensures on 5 hues; we only have 3, so this change is not necessary.